test/e2e/node/pods.go

```diff
@@ -379,12 +379,14 @@ var _ = SIGDescribe("Pods Extended", func() {
 								switch {
 								case t.ExitCode == 1:
 									// expected
+								case t.ExitCode == 137 && (t.Reason == "ContainerStatusUnknown" || t.Reason == "Error"):
```

Comment:
Did this start happening with your PR?

Reply:
If so, was it because previously we would have left the container in waiting state during graceful shutdown?

Reply:
> Did this start happening with your PR?

Yes.

> If so, was it because previously we would have left the container in waiting state during graceful shutdown?

Either waiting or running I think. Statuses weren't properly getting reported.